nservicebus/azure-service-bus/namespace-hierarchy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Namespace hierarchy support
+summary: Configuring Azure Service Bus transport to support namespace hierarchies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+---
+
+At the core of the Azure Service Bus service, there is a Service Registry which tracks the location of each queue, topic, relay or eventhub in the service. This Service Registry provides a DNS integrated hierarchical naming system, that has a root entry point (called a namespace) at a uri with the following scheme.
+
+`https://{serviceNamespace}.servicebus.Windows.net/{path}`
+
+The {path} parameter can be specified at any depth, allowing the hierarchy to be extended with application specific sub trees. This is very usefull for large systems to group, manage and secure resources more efficiently.
+
+for example:
+
+```
```

Comment:
can u append a `no-highlight`. if not hljs will guess the langauge and it is not too good at it
more info: https://github.com/Particular/docs.particular.net#inline-code